code/segmentedtickerplant/stplog.q

```diff
@@ -102,7 +109,9 @@ zts[`defaultbatch]:{
 
 // Immediate mode - publish and write immediately
 upd[`immediate]:{[t;x;now]
-  x:updtab[t] . (x;now);
```

Comment:
Instead of modifying the upd function, if in ctp mode, make updtab just have a default definition for each table which just returns the raw table as is e.g. updtab[`trade]:{[x;now] x}